python/cuml/cuml/explainer/kernel_shap.pyx

```diff
@@ -194,7 +192,7 @@ class KernelExplainer(SHAPBase):
                  random_state=None,
                  is_gpu_model=None,
                  handle=None,
-                 dtype=None,
+                 dtype="float32",
```

Comment:
We state that valid values are `np.float32` or `np.float64`, but then accept "float32" and "float64" and set the default to "float32". I think we should set the default to `np.float32`, but still accept the string format.